wiki/Website-Media.md

```diff
@@ -41,3 +43,44 @@ Regarding the format and settings of the "Export" panel in Figma, please follow
   - is it an image that resizes when the viewport changes? speak with the developer and decide what is the right size to use
 
 If you have any doubts or problems exporting the files, speak with one of the designers of the HDS team.
+
+## Optimizing images
+
+Prior to adding new images to the website the assets should be optimized to reduce their file size and increase our web performance. Figma's export feature doesn't allow for much fine-grained control regarding image quality and optimizing for speed on the web. [ImageOptim](https://imageoptim.com) works great, is free, easy to use, and with the correct settings can result in a more than _50% reduction_ in file size per image.
```

Comment:
```suggestion
Prior to adding new images to the website, the assets should be optimized to reduce file sizes and increase web performance. Figma’s export feature doesn’t allow for fine-grained control regarding image quality and optimization for the web. [ImageOptim](https://imageoptim.com) is free, works great, and is easy to use. And with the correct settings, it can result in a more than _50% reduction_ in file size per image.
```